src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const marriedON = sex === 'm'
+    ? 'wife'
+    : 'husband';
+  const personAge = age
+    ? <p className="Person__age">{`I am ${age}`}</p>
+    : undefined;
```

Comment:
IMO, it's better to do all conditional rendering inside JSX, because now I need to jump between variables and remember what will be rendered and where 